src/utils/gulp/gulp-tasks-test.js

```diff
@@ -17,38 +21,47 @@ module.exports = function(gulp, options) {
   };
 
   gulp.task('test', function(done) {
-    if (options.testPaths) {
```

Comment:
If we decide to move away from `testPaths` we need to add backward compatibility, meaning we should deprecated the options while still supporting it for now.

But, I believe we can maintain `options.testPaths` as is.